settings.gradle

```diff
@@ -1 +1,5 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
```

Comment:
Can you help me understand which problem exactly we are solving by introducing this?

Reply:
Sure, that ensures that if the toolchain needed for the project is missing, then it gets installed.
See here for the details: https://docs.gradle.org/current/userguide/toolchains.html#sub:download_repositories
I didn't have the right one for some reason, and that stanza was solving that.

Reply:
Ok, sounds reasonable